src/globus_sdk/services/compute/client.py

```diff
@@ -236,6 +236,69 @@ class ComputeClientV3(client.BaseClient):
     scopes = ComputeScopes
     default_scope_requirements = [Scope(ComputeScopes.all)]
 
+    def register_endpoint(self, data: dict[str, t.Any]) -> GlobusHTTPResponse:
+        """Register a new endpoint.
+
+        :param data: An endpoint registration document.
+
+        .. tab-set::
+
+            .. tab-item:: API Info
+
+                .. extdoclink:: Register Endpoint
+                    :service: compute
+                    :ref: Endpoints/operation/register_endpoint_v3_endpoints_post
+        """
+        return self.post("/v3/endpoints", data=data)
+
+    def update_endpoint(
+        self, endpoint_id: UUIDLike, data: dict[str, t.Any]
+    ) -> GlobusHTTPResponse:
+        """Update an endpoint.
+
+        :param endpoint_id: The ID of the endpoint.
+        :param data: An endpoint update document.
+
+        .. tab-set::
+
+            .. tab-item:: API Info
+
+                .. extdoclink:: Update Endpoint
+                    :service: compute
+                    :ref: Endpoints/operation/update_endpoint_v3_endpoints__endpoint_uuid__put
+        """  # noqa: E501
+        return self.put(f"/v3/endpoints/{endpoint_id}", data=data)
+
+    def lock_endpoint(self, endpoint_id: UUIDLike) -> GlobusHTTPResponse:
+        """Temporarily block registration requests for the endpoint.
+
+        :param endpoint_id: The ID of the Globus Compute endpoint.
+
+        .. tab-set::
+
+            .. tab-item:: API Info
+
+                .. extdoclink:: Delete Endpoint
```

Comment:
This doesn't look right; it's in the `lock_endpoint()` method.

Reply:
Ah, good catch.